cms/static/js/spec/views/move_xblock_spec.js

```diff
@@ -0,0 +1,277 @@
+                return {
+                    categoryText: viewEl.find('.category-text').text().trim(),
+                    currentLocationText: viewEl.find('.current-location').text().trim(),
+                    xblockCount: viewEl.find('.xblock-item').length,
+                    xblockDisplayNames: viewEl.find('.xblock-item .xblock-displayname').map(
+                        function() { return $(this).text().trim(); }
+                    ).get(),
+                    forwardButtonSRTexts: viewEl.find('.xblock-item .forward-sr-text').map(
+                        function() { return $(this).text().trim(); }
+                    ).get(),
+                    forwardButtonCount: viewEl.find('.fa-arrow-right.forward-sr-icon').length
+                };
+            };
+
+            verifyListViewInfo = function(category, expectedXBlocksCount, hasCurrentLocation) {
+                var displayedInfo = getDisplayedInfo();
+                expect(displayedInfo.categoryText).toEqual(moveXBlockListView.categoriesText[category] + ':');
+                expect(displayedInfo.xblockCount).toEqual(expectedXBlocksCount);
+                expect(displayedInfo.xblockDisplayNames).toEqual(
+                    _.map(_.range(expectedXBlocksCount), function(xblockIndex) {
+                        return category + '_display_name_' + xblockIndex;
+                    })
+                );
+                if (category !== 'component') {
+                    if (hasCurrentLocation) {
+                        expect(displayedInfo.currentLocationText).toEqual('(Current location)');
+                    }
+                    expect(displayedInfo.forwardButtonSRTexts).toEqual(
+                        _.map(_.range(expectedXBlocksCount), function() {
+                            return 'Press button to see ' + category + ' childs';
+                        })
+                    );
+                    expect(displayedInfo.forwardButtonCount).toEqual(expectedXBlocksCount);
+                }
+            };
+
+            verifyBreadcrumbViewInfo = function(category, xblockIndex) {
+                var displayedBreadcrumbs = moveXBlockBreadcrumbView.$el.find('.breadcrumbs .bc-container').map(
+                    function() { return $(this).text().trim(); }
+                ).get(),
+                    categories = _.keys(parentToChildMap).concat(['component']),
+                    visitedCategories = categories.slice(0, _.indexOf(categories, category));
+
+                expect(displayedBreadcrumbs).toEqual(
+                    _.map(visitedCategories, function(cat) {
+                        return cat === 'course' ?
+                            'Course Outline' : cat + '_display_name_' + xblockIndex;
+                    })
+                );
+            };
+
+            clickForwardButton = function(buttonIndex) {
+                moveXBlockListView.$el.find('[data-item-index="' + buttonIndex + '"] button').click();
```

Comment:
Let's add the default value `buttonIndex=buttonIndex || 0` and remove parameter `0` from all callers.